torch_ort_inference/docker/README.md

```diff
@@ -0,0 +1,53 @@
+# OpenVINO™ integration with PyTorch ONNX Runtime Dockerfiles for Ubuntu* 18.04 and Ubuntu* 20.04
+
+
+We provide Dockerfiles for Ubuntu* 18.04 and Ubuntu* 20.04 which can be used to build runtime Docker* images for OpenVINO™ integration with PyTorch on CPU, iGPU and VPU.
```

Comment:
OpenVINO™ integration with PyTorch change to "OpenVINO™ integration with Torch-ORT"

Reply:
fixed